spec/lib/embed/viewer/geo_spec.rb

```diff
@@ -50,13 +50,13 @@
     end
     it 'generates a file list when file has resources' do
       stub_purl_response_and_request(geo_purl_public, request)
-      html = Capybara.string(geo_viewer.download_html)
+      html = Capybara.string(geo_viewer.download_html.to_html)
```

Comment:
there's something uncomfortable to me about needing to call "to_html"  on a method called download_html.   Should the object be named download_panel?   Should "to.html" be called inside   download_html method?    Or is my ignorance showing?